src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (callback) {
+    let sortFunction = callback;
+
+    if (sortFunction === undefined) {
+      sortFunction = (item1, item2) => {
+        const string1 = String(item1);
+        const string2 = String(item2);
+
+        if (string1 > string2) {
+          return 1;
+        } else {
+          return -1;
+        }
+      };
+    }
```

Comment:
looks redundant, just specify the default value for callback
```suggestion
  [].__proto__.sort2 = function (callback = ...) {
```